README.md

```diff
@@ -138,13 +137,10 @@ spec:
   - the `prefix` is optional and must be a DNS subdomain: a series of DNS labels separated by dots `.`,
     it must contain at most 253 characters, followed by a slash `/`.
   - the `openslo.com/` is reserved for OpenSLO usage
-- **indicator** optional, represents the Service Level Indicator (SLI).
-  Currently this only supports one Metric, `thresholdMetric`, with `ratioMetrics`
-  supported in the [objectives][15] stanza.
-- **indicator.thresholdMetric** *Metric*, represents the query used for
-  gathering data from metric sources. Raw data is used to compare objectives
-  (threshold) values. If `thresholdMetric` is defined then `ratioMetrics`
-  should be excluded in [objectives][16].
+- **indicator** optional, represents the Service Level Indicator (SLI),
+  described in [SLI](#sli) section.
+- **indicatorRef** optional, this is the name of Service Level Indicator (SLI).
```

Comment:
Would this be the name of the file or the name (value of the `metadata.name`) of the SLI?

Reply:
I think it should align with the existing **ref** fields for consistency purposes.

@weyert  do you happen to know whether these two existing ref fields refer to *the name of the file* or *the name of the SLI*? The definitions seems vague to me.

* conditionRef string: this is the name or path the Alert condition
* targetRef string: this is the name or path the Alert Notification Target

Reply:
The Kubernetes pattern is `metadata.name` is used for any `ref` fields. I'd stick with that

Reply:
@weyert I agree with @kenfinnigan's opinion. 

We can either write all SLIs in one file or separate in multiple files. In whichever case, the tooling should be able to resolve to the designated SLI by name.

While having path has some issues. If there are two SLIs defined in a file in the following form, then name of the file cannot indicate a single referenced indicator.

```yaml
---
- kind: SLI
   ...
- kind: SLI
   ...
```

Reply:
Sorry, totally missed this conversation. My originally idea was that you could specify the name or a direct reference to the file itself. But I didn't think off the possibility that you could define  multiple ones in one YAML file. Thank you for pointing that out. Personally, I like to put it in separate files so didn't think of it :D

I don't have strong feelings about this, if by name makes sense, and is considered 'standard' or 'common' in the Kubernetes world. I think it would be fine.

Reply:
Thanks for the feedback. I'll stick with the given statement.